internal/pkg/build/sources/base_environment.go

```diff
@@ -123,26 +124,26 @@ fi
 `
 	// Contents of /.singularity.d/env/01-base.sh
 	baseShFileContent = `#!/bin/sh
-# 
```

Comment:
Even if the trailing spaces are incorrect, please try to avoid changing it in this PR, to keep the diffs clean.

Reply:
Sorry, my editor just corrected the whitespace automatically.  I'll revert the change.

Reply:
OK, I've fixed the CI/formatting and whitespace issues.  Let me know if you'd like me to squash the commits.

Reply:
I've just squashed the commits as it's cleaner in my opinion.